jnigen/lib/src/writers/files_writer.dart

```diff
@@ -24,6 +24,7 @@ String getFileClassName(String binaryName) {
 /// is mirrored.
 class FilePathResolver implements SymbolResolver {
   FilePathResolver(
+    this.classes,
```

Comment:
Please make these named arguments, the call-sites are very hard to understand.

Also, consider making them optional (just default to `const {}`).